_pages/detailed-process.md

```diff
@@ -119,14 +110,15 @@ Notes on interview format:
     - panel interviews are intimidating, and can magnify mistakes (candidates don't get another "shot" to recover)
     - exception: "training" interviewers may sit in to observe, but if so they should be muted and video off, just an observer
 - interviewers should ask questions off relevant guides, no one-offs
-    - overlap with previous interviews is just fine (even, good - gives a candidate a chance to recover )
+    - overlap with previous interviews is just fine (even, good - gives a candidate a chance to recover)
 
 ### 6. Decision time
 
-- Interviewers submit feedback in writing to TL+HM (see guide below for format)
-- If the results are especially controversial, hold a meeting to discuss
-- HM makes final hire/no-hire decision
-    - Consensus isn't required, nor is there any voting. The HM makes the final call.
+- Interviewers submit feedback to TL+HM using the Google Form that TL attaches to your interview event.
```

Comment:
We do define TL and HM higher up in the doc, right?

Reply:
:+1: Yep, we do.